packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -237,64 +266,72 @@ export const SchemaDrivenAnnotationEditor = (
               } as JSONSchema7
             }
             uiSchema={{
-              'ui:DuplicateKeySuffixSeparator': '_',
+              'ui:options': {
+                copyable: true,
```

Comment:
New feature in RJSF v5.x - array items are copyable. Might as well surface this to our users!